macos/FluentUI/Link.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+
+/// A lightweight hyperlink control
+@objc(MSFLink)
+open class Link : NSButton {
```

Comment:
should it be linkButton?

Reply:
I wanted to clearly distinguish it from being a button, since [FluentUI React Native](https://developer.microsoft.com/en-us/fluentui#/controls/crossplatform) has them as separate controls. Also reserve the right to reimplement this with NSTextView or something else if we find it's better off that way 🙂